src/app/src/components/Navigation.tsx

```diff
@@ -2,18 +2,19 @@ import React, { useState } from 'react';
 import type { LinkProps } from 'react-router-dom';
 import { Link, useLocation } from 'react-router-dom';
 import { IS_RUNNING_LOCALLY } from '@app/constants';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
 import EngineeringIcon from '@mui/icons-material/Engineering';
 import InfoIcon from '@mui/icons-material/Info';
+import { AppBar, Toolbar, Button, IconButton, Menu, MenuItem, Box, Tooltip } from '@mui/material';
```

Comment:
we have a convention of expanding these imports for tree shaking/dev experience

https://mui.com/material-ui/guides/minimizing-bundle-size/#development-environment

I think we violated this in the redteam setup, so we need to go back and fix that, but at least we shouldn't spread it to other files